packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -163,7 +163,7 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
   return (
     <>
       <Navigate />
-      <Title />
+      <TitleElement />
```

Comment:
should `TitleControl` be used via [ControlsContextProvider](https://github.com/aws-amplify/amplify-ui/blob/feat-storage-browser/main/packages/react-storage/src/components/StorageBrowser/views/LocationsView/LocationsView.tsx#L111) on all the views.

The props the component takes are passed via ControlsContextProvider and the component itself only takes className.